django_lti_tool_provider/migrations/0002_reduce_custom_key_length.py

```diff
@@ -32,8 +25,23 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.RunPython(
-            code=check_field_max_length_lteq_190,
-            reverse_code=reverse_migration
+            code=check_field_max_length_lteq_190
+        ),
+        migrations.RunSQL(
+            sql=migrations.RunSQL.noop,
+            state_operations=[
+                # This is a shim. Django is oh-so-efficient, so if it detects
+                # that the database should be in a state based on previous migrations,
+                # it won't perform the operations to move to that state. This tells
+                # Django "hey, I'm in this different state" without actually modifying
+                # the database. This is not a reversible operation.
```

Comment:
Please also mention somewhere in this file that the whole reason for this is that the old version of migration `0001` set `max_length=400`, but the new version sets `max_length=190`, so this migration is weird in order to make those disparate states consistent.